python/requirements-wheel-test.txt

```diff
@@ -1,10 +1,7 @@
 cffi
 cython
 hypothesis
-numpy==1.14.5; python_version < "3.7"
-numpy==1.16.0; python_version == "3.7"
-numpy==1.17.3; python_version == "3.8"
-numpy==1.19.3; python_version >= "3.9"
+numpy==1.19.4
```

Comment:
@xhochy shouldn't we test with 1.16.6 for python < 3.9 instead?

Reply:
We should test with the `numpy` release we have built with and a much newer one to detect issues. Currently we only test with one and there I would prefer to test with a newer release as this has a higher chance to fail on the C-API level like we have seen with the 1.20 release candidate.

Reply:
Ok, sounds good to me.